webcompat/form.py

```diff
@@ -65,6 +67,9 @@
 browser_test_label = u'Did you test in another browser?'
 textarea_label = u'Please describe what happened, including any steps you took before you saw the problem'  # noqa
 
+contact_message = u'There is a mistake in the username.'  # noqa
```

Comment:
I think we probably need to make similar / duplicate changes in bugform.js to be able to react to this before form submission.

Reply:
See the following for some hints on that

https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/js/lib/bugform.js#L25-L28
https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/js/lib/bugform.js#L313-L320
https://github.com/webcompat/webcompat.com/blob/master/webcompat/static/js/lib/bugform.js#L355

Reply:
@miketaylr are we duplicating in bugform.js or in templates what should be done in one place. Do we have an opportunity to remove in one of the two places?

Reply:
@miketaylr ok I added a couple of things here. I think the helptext is not working but I'm not sure why.

https://github.com/webcompat/webcompat.com/pull/2713/commits/3a261f8d6216b873001ec1e7c5a78f714fc595ab

Once this is right I can add a test on the functional side.

Reply:
> Ah indeed my validation code is pretty much useless here. Given that we do not do anything with it server side, where we would check if the field was valid or not and have an action about it.

It used to be the case that server-only validation would work (before we added all the client side stuff). I'm not 100% sure if it does, because we don't really exercise that code path. But the theory is WTForm will catch the validation errors and Flask will re-render the page using the errors that WTForm gives it:

```
{% if form.contact.errors %}
  {% for error in form.contact.errors %}
  <small class="form-message-error">{{ error }}</small>
  {% endfor %}
{% endif %}
```